### ISSUE TITLE
What kind of proposals are we looking for? 

### ISSUE THREAD

**Issue by arojascastro** [programminghistorian/jekyll#751]
I do not intend to create a never-ending discussion, however, after all the issues that we have recently faced, I am wondering if we should rethink our definition of "ideal" lesson. This translate in a very specific way in our Guidelines for authors, where we state: 

> There is no standard lesson at the Programming Historian. We welcome lessons on any topic that helps humanists at any phase of the research and publication process. // We welcome all lesson proposals, pitched at any level of technical aptitude and experience, from neophytes to power users. Lessons may be a short and straightforward explanation of a discrete task (such “helper” lessons make complex lessons easier to follow); they may be quite long, complex, and technically sophisticated.

See the context [here](https://programminghistorian.org/author-guidelines#proposing-a-new-lesson).

We should be able to be more concrete and say explicitly what are we looking for. From my point of view the following characterstics are essential:

- relevant for the humanities
- sustainable in the long term
- short and focused on one problem, process or task
- international and language independent (or for a global audience)

...

Is there any more traits that can we collect from all the policies and discussions that are we having? Maybe inclusive (in gender and racial terms)?

Do you think is it worth discussing this topic? Or should we refrain from establishing a standard lesson?

**Comment by drjwbaker** [comment]
I take..

> There is no standard lesson at the Programming Historian et cetera

.. as a statement about the **thematic** content of the lessons we are looking for, which is separate from - say - the issues of sustainability or language independent which, @arojascastro, you rightly point out we also want. Maybe it is a case of making that explicit, so that sustainability or language independent don't appear to come as secondary considerations?

**Comment by arojascastro** [comment]
Ah, I missed that point... so if it means *thematic* it makes sense - though not all topics should be accepted, only those relevant for the humanities, I think (or maybe I'm wrong). 

But after that, we can read things about:

- length 
- level of expertise

I argue for saying something extra, but only if that would help authors and reviewers.

**Comment by drjwbaker** [comment]
@arojascastro No, I think you are correct: the *thematic* nature of that text is implied rather than explicit.

**Comment by arojascastro** [comment]
On the other hand, maybe we can be a little bit more concrete about methods? Let's have a look at our index:

![captura de pantalla 2018-03-21 a las 12 13 47](https://user-images.githubusercontent.com/7340111/37706883-5c5c25c8-2d01-11e8-85f1-b599d10767e7.png)

**Comment by JMParr** [comment]
Good discussion. Is this worth adding to our agenda for next week's editorial call?

**Comment by walshbr** [comment]
Added it to the agenda. For what it's worth, anyone can add items to the agenda by editing the comment I've made at the top of the editorial meeting ticket - you can then edit the text of the agenda directly.

**Comment by arojascastro** [comment]
Since it seems there is no further discussion and we did not reach any agreement, I think we can close this issue and reopen later if someone else wants to contribute.

**Comment by drjwbaker** [comment]
Just adding as a note that one objection to one part of this - having length criteria - is that we have no evidence that length criteria has an pedagogical value. If we find, however, some pedagogical lit that suggests we should consider length criteria, that might be one reason to reopen this.

**Comment by arojascastro** [comment]
The point is not being pedagogical. The point is to avoid extra work during the reviewing process for the reviewer and for the author, who will have to rewrite the text. You can check out this ticket review where  @nolauren suggested to split the tutorial into two: https://github.com/programminghistorian/ph-submissions/issues/102#issuecomment-378630029 (I believe it is not an isolated case).

On the other hand, there are more criteria apart from lenght. It does not make any sense to demand sustainability or cosmopolitalism/internationalism if we do not state that explicitly. These are characteristics that authors need to know in advance before starting to write.

**Comment by acrymble** [comment]
@drjwbaker there's also a cost £££ to extra long papers if we are paying for copy editing.

**Comment by drjwbaker** [comment]
@arojascastro Okay. How do you want to proceed? Make it part of another issue? Reopen with a concrete proposal or action plan? Set a date to revisit and review? (we could - for example - agree to annually review our high level text on the type of lessons we are looking for)

**Comment by arojascastro** [comment]
I would simply modify the text: 

> There is no standard lesson at the Programming Historian. We welcome lessons on any topic that helps humanists at any phase of the research and publication process. We welcome all lesson proposals, pitched at any level of technical aptitude and experience, from neophytes to power users. Lessons may be a short and straightforward explanation of a discrete task (such “helper” lessons make complex lessons easier to follow); they may be quite long, complex, and technically sophisticated.

to 

> Although there is no standard lesson at the Programming Historian, we welcome tutorials relevant for the humanities, pitched at any level of technical aptitude and experience, from neophytes to power users, that focus on one problem or process, can be sustainable in the long term and are addressed to a global audience.

Something like that. It is not a big change, I think. There are other places in the guidelines where we could suggest something similar. 

However, I would only change this if there is consensus, which I think is not the case.

**Comment by drjwbaker** [comment]
Okay. Thing is, the way I read the flow of the paragraphs at https://programminghistorian.org/author-guidelines#proposing-a-new-lesson is that they proceed in 4 parts after the highlighted 'There is no standard lesson..' text:

- paragraph on possible content of a lesson
- paragraph on example lessons
- paragraph on expected sustainability of a lesson
- paragraph on process of lesson publishing

So rather than edit the first paragraph (as you suggest above), it strikes me that we need a proper paragraph on internationalisation in that top section (a 5th part). This could be a useful outcome of the work at #768. Does this work for you @arojascastro?

**Comment by arojascastro** [comment]
I already authored two proposals about this issue. I am not writing anymore because as a non native English speaker it takes me longer to write and I am not in the mood at the moment to submitt a text that it is not going to meet your needs or expectations. I think other people should think about and contribute.

**Comment by drjwbaker** [comment]
@arojascastro I'm sorry you feel frustrated. What you propose does meet my needs and expectations. In fact, I think you are right: internalisation is missing from these guidelines and that has become clear in recent months as issues translating lessons has arisen. I am merely pointing out what I see as the logic of this section of text as it stands at present and proposing a solution that - as far as I can see - does what you want (add some wording on our desire for lessons to address a global audience) without unpicking the logic we have.

**Comment by acrymble** [comment]
I like @arojascastro 's suggested text change. I think we should adopt it.

**Comment by walshbr** [comment]
I agree - I like the suggested change. I'm sorry we didn't have more time to discuss it in the call last week as well @arojascastro - we can definitely put it back on the agenda, though I think what you've suggested here makes good sense.

**Comment by acrymble** [comment]
For discussion at meeting. I propose adjusting @arojascastro 's text to:

We welcome tutorials relevant for the humanities, pitched at any level of technical aptitude and experience, from neophytes to power users, that focus on one problem or process, can be sustainable in the long term and are addressed to a global audience. Tutorials should not exceed 8,000 words including code, and we expect that most lessons will be between 4,000 and 6,000 words.

Linked to issue #779

**Comment by drjwbaker** [comment]
On..

> including code

..is that strictly necessary? I presume - perhaps incorrectly - that code is not translated, and although we welcome crisp code, it is likely a requirement we will waive if the code needs to be long for some reason, no?

**Comment by acrymble** [comment]
Ya they have been translating code. Variables, strings, etc. Also a lot easier to do a word count on (otherwise you have to delete all the code). It's just a suggestion for a starting point. Only 4 out of 100 lessons wouldn't come in under that count.

**Comment by drjwbaker** [comment]
Okay. My bad. Keep it in then. Maybe we need - in due course - to make the fact that code is translated clear to authors/editors/reviewers in our docs.

**Comment by amsichani** [comment]
OK, as far as I can see, a need has been expressed to review different aspects of the author/ reviewers guidelines, but this process could encompass multiple needs and speeds.
I am grouping here a number of ongoing discussions/issues on two distinct groups:

1. straightforward ones - amending the paragraph in the author guidelines

-   lessons' level of expertise: I think we shouldn't omit the "Lessons may be a  straightforward explanation of a discrete task (such “helper” lessons make complex lessons easier to follow) ; they may be complex, and technically sophisticated".  (I have omitted though the 'short-long' terms, in order to describe the...)
-  lessons' length <a href="https://github.com/programminghistorian/jekyll/issues/779">#779</a> . @acrymble  suggested : " Tutorials should not exceed 8,000 words including code, and we expect that most lessons will be between 4,000 and 6,000 words." 
-  mentioning briefly the sustainability and internationalisation as principles of our tutorials

The paragraph in question could look like this  :
"*We welcome tutorials relevant for the humanities, pitched at any level of technical aptitude and experience, from neophytes to power users, that focus on one problem or process, can be sustainable in the long term and are addressed to a global audience.* Lessons may be a straightforward explanation of a discrete task (such “helper” lessons make complex lessons easier to follow) ; they may be complex, and technically sophisticated. Tutorials should not exceed 8,000 words including code, and we expect that most lessons will be between 4,000 and 6,000 words."
 [we can put the text inside the * * as highlighted]

2. need more work - ones
- internationalisation <a href="https://github.com/programminghistorian/jekyll/issues/768"> #768 </a> & language independence <a href="https://github.com/programminghistorian/jekyll/issues/651"> #651 </a>
These are two related topics but, in my view, not  identical. I think its a happy coincidence that PH came across the issue of internationalisation through the translation efforts of the Spanish team.  Having an international, non-cultural/language specific set of guidelines for our lessons, is, in my eyes, a major milestone for designing an inclusive pedagogical output. It will, of course, assist any future translation endeavour, but its tremendously invaluable on its own. Especially given the recent facts of our growing and highly-diversed audience (India), I think this should be among our first priorities in terms of policy - strategy. 

But internationalisation should be seen as something distinct from the easiness to translate a lesson. Internationalisation as a principle should apply to all PH original lessons (English, Spanish or French or even Greek in the future :-) The language independence as such is a different principle and I am not sure whether there is a need to draft a firm strategy for it in the long term. An original Spanish lesson should of course be in line with our internationalisation principles, but even if is not easy to translate in English, this doesn't mean it isn't useful for Spanish readers - and vice versa. I am also aware that language independence as a firm 'rule' could also warm our authors' willingness to contribute as they might  feel insecure about their bibliography, sources etc. **We should celebrate and promote diversity, even here, and in my view language independence as a firm strategy ceases originality in the sake of uniformity**. Instead of drafting a set of rules  and fighting for language independence, its more clever if we amend the reviewer guidelines so the reviewers will ensure that the lesson acknowledges language/material -specific dependencies etc (we could create a list of issues to check). Eg. coming myself from a country/ language (Greece) with closely to 0 DH teaching resources, I found it extremely important to have tutorials on methods or software in other languages that even if there are not using Greek primary sources or secondary literature, at least they are not hostile to my scholarly and cultural repertoire, to the non-latin character set et.c.

I want, thus, to give back to @arojascastro his somehow-lost enthusiasm on the topic. I think his draft text on <a href="https://github.com/programminghistorian/jekyll/issues/651#issuecomment-363722402"> 'internationalization and localization to our guidelines for authors, editors and reviewers'</a> should be our starting point for such an endeavour and we could creatively work towards a set of internationalisation guidelines and taking care for language or material dependencies of lessons. This process and its application might take a bit longer than just drafting a straight forward short paragraph (aka 1), but I 'm sure it will worth it ;-) [ sorry for the long comment -;]

**Comment by arojascastro** [comment]
As I said before, I suggest other people to write a proposal or adapt mine - this and also issue #651. 

I agree with Anna Maria that this paragraph is just part of the solution. I like Adam's additions and corrections. 

I see the point of Anna's about internationalization / translation, they are different topics but they are very related in my opinion - you cannot separate them when translating. I recommend reading about GILT and localization. There is a lot of research done. Here is one article (this is not a scholarly article but there are lot): https://blog-english.jrlanguage.com/what-is-gilt-globalization-internationalization-localization-and-translation/

In all the research that I have read, internationalization is the previous stage to enable translation and localization. But of course even if the lessons were not going to be translated it would be great to have an internationalization strategy for its own sake.

I do not agree much with this pararaph: "An original Spanish lesson should of course be in line with our internationalisation principles, but even if is not easy to translate in English, this doesn't mean it isn't useful for Spanish readers - and vice versa. I am also aware that language independence as a firm 'rule' could also warm our authors' willingness to contribute as they might feel insecure about their bibliography, sources etc. We should celebrate and promote diversity, even here, and in my view language independence as a firm strategy ceases originality in the sake of uniformity. Instead of drafting a set of rules and fighting for language independence, its more clever if we amend the reviewer guidelines so the reviewers will ensure that the lesson acknowledges language/material -specific dependencies etc (we could create a list of issues to check)." 

Basically, because lessons in Spanish have to work and some tools do not work with Spanish texts. Of course they are worth reading, but then Spanish readers have the original. That's why we are not translating them into Spanish. It makes no sense. Maybe we should listen to other people that have edited or translated like @mariajoafana, @vgayolrs @jenniferisasi or @jamotilla 

I strongly defend including recommendations on internationalization on our guidelines simply because authors may shape their contribution in a different way if they were awared of this. At the moment they are not aware at all. Also I insist that reviewers usually focus on tecnical aspects, I guess because there is no metion to internationalization and localization on our guidelines. 

I have jumped into twice reviewing processes and suggested changes. I think they were well welcomed in both cases and I do no think that was damaging at all. I do no think it is a big deal and we shouldn't be afraid of. Hower, I cannot and I do not want to continue jumping in reviewing processes of English lessons. So I will leave this discussion on your hands. I will continue claiming that we need to expand our guidelines and put more effort on internationalization from the beginning of the pipeline - not only at the end when the text is translated. If you need help, let me know.

**Comment by amsichani** [comment]
I hear @arojascastro 's position on the need to have internationalisation as a claim both for a robust set guidelines but also as an ethos embedded on our lessons' review processes that are now on their way. I suggest we could adopt an interim strategy for these specific cases: as Antonio has genuinely jumped to two reviewing cases of English lessons, I offer myself to do so in the meantime .
I strongly believe too that we need to develop a strong piece on internationalisation on our guidelines, as one of our top and more urgent policy priorities. Happy to contribute on this.

**Comment by acrymble** [comment]
@mariajoafana and I are hosting a workshop this summer in Bogota, and would love to include a session on writing for an international audience. So if we can get some resources in place, we can start building it into our training.

**Comment by amsichani** [comment]
I think its doable (when is your workshop?) and a good place/way to test also these guidelines. I am thinking the best way is to go back to #651  and work on it. @arojascastro et al. had put a lot of effort and great stuff there.

**Comment by acrymble** [comment]
@amsichani the workshop is 1-3 August.

**Comment by drjwbaker** [comment]
I agree with the position we have reached in this ticket.

**Comment by arojascastro** [comment]
plus 1

**Comment by jenniferisasi** [comment]
I agree too.

**Comment by acrymble** [comment]
This involves amending the same files as currently being amended for issue #651 so I will change this once that ticket has been closed.
